src/ztc/api/tests/test_besluittype.py

```diff
@@ -1,30 +1,26 @@
-from ...datamodel.tests.factories import BesluitTypeFactory
+from rest_framework import status
+
+from ...datamodel.models import BesluitType
+from ...datamodel.tests.factories import (
+    BesluitTypeFactory, InformatieObjectTypeFactory, ZaakTypeFactory
+)
 from .base import APITestCase
 from .utils import reverse
 
 
 class BesluitTypeAPITests(APITestCase):
```

Comment:
can you add a test where you try to relate it to objects from a different catalogue, which should fail?

Reply:
additional tests for `besluittype` and `zaaktype` creation are added